packages/ListBoxWithTags/src/useListBoxWithTags.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import { filter } from "./helpers";
+
+function prepareDataDictionary(key, data) {
+  const dictionary = {};
+
+  data.forEach(d => {
+    dictionary[d[key]] = d;
+  });
+
+  return dictionary;
+}
+
+export default function useListBoxWithTags(
+  key,
```

Comment:
What do you think about putting `key` in the object, too?

Reply:
I thought it for the past two hours and agree makes sense, will be more descriptive having the word key on the object.